### DISCUSSION TITLE
How can I add headers into useFetch options or $fetch

### DISCUSSION THREAD

**Discussion by congyaqwq** [nuxt/nuxt#16233] [Questions]
There are some default headers  when I add nothing  into options's headers
like
<img width="977" alt="Screen Shot 2021-11-29 at 5 34 28 PM" src="https://user-images.githubusercontent.com/45482161/143843123-674d8b06-28c2-43d8-be62-a43a4961b59e.png">
  
But when I tried to add some headers
```js
useFetch(url, {
      baseURL,
      params,
      headers: {
        withCredentials: true,
      },
      ...options,
    });
```

other default headers disappear.  
<img width="940" alt="Screen Shot 2021-11-29 at 5 36 34 PM" src="https://user-images.githubusercontent.com/45482161/143843555-0c0f0cdb-dffe-481e-981e-438948dd4251.png">

**Comment by hermesalvesbr** [comment]
Did you manage to solve it?

**Reply by congyaqwq** [reply]
No, I need it for my personal development blog, so it's on hold;

**Comment by sangemi** [comment]
(plus)+ I want to set default headers (jwt token)
